CHANGELOG.md

```diff
@@ -62,6 +62,9 @@
 - [#4080](https://github.com/ChainSafe/forest/pull/4080) Fix broken
   `StateVMCirculatingSupplyInternal` RPC method on calibnet.
 
+- [#4085](https://github.com/ChainSafe/forest/pull/4085) Fix discrepancies in
```

Comment:
`!Filecoin.StateCall` needs to be removed from `filter-list` file as well

Reply:
Done. Thanks!